demos/storybook/stories/drawer/_module.stories.ts

```diff
@@ -112,6 +112,5 @@ storiesOf(`${COMPONENT_SECTION_NAME}/Drawer`, module)
     .addDecorator(withA11y)
     .addDecorator(storyWrapper())
     .addParameters({ ...STORY_PARAMS, notes: { markdown: getReadMe('Drawer.md') } })
-    .add(README_STORY_NAME, getReadMeStory)
```

Comment:
Curious why this got removed?

Reply:
https://github.com/pxblue/angular-component-library/pull/299#issuecomment-869987516

Reply:
I noticed some components still have it and some don't...we should probably have a cleanup issue to go in and remove them all

Reply:
This instance is a special case because the Drawer stories have 2 modules defined.  The README story was duplicated between these two modules so one instance had to removed.  All the other components should be fine.